frames/src/main/java/com/checkout/frames/style/theme/PaymentFormTheme.kt

```diff
@@ -0,0 +1,24 @@
+package com.checkout.frames.style.theme
+
+public data class PaymentFormTheme(
+    /**
+     * Custom colors provided by the merchants
+     */
+    val paymentFormThemeColors: PaymentFormThemeColors,
+    /**
+     * Custom payment form fields used by the merchants
+     */
+    val paymentFormComponents: List<PaymentFormComponent>,
```

Comment:
It would be nice to set all components by default, so merchants can just change the color scheme without components set up.